### PR TITLE
chore: update rbac plugins to use the migrated backstage-community ones

### PR DESCRIPTION
**Pull request by 04kash**
## Description

updated rbac plugins to use the migrated backstage-community ones

## Which issue(s) does this PR fix

- Fixes: https://issues.redhat.com/browse/RHIDP-4766 

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by AndrienkoAleksandr — COMMENTED** [review]
Looks good to me. Tested - works fine. LGTM when e2e will be green.

**Review by AndrienkoAleksandr — APPROVED** [review]

File: dynamic-plugins/wrappers/backstage-community-plugin-ocm/turbo.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "tasks": {
     "tsc": {
       "outputs": [
-        "../../../dynamic-plugins/wrappers/backstage-community-plugin-ocm/**"
+        "../../../dist-types/dynamic-plugins/wrappers/backstage-community-plugin-ocm/**"
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-community-plugin-ocm/turbo.json:6]: why this change?

**Comment by nickboldt** [comment]
Olex approved, LGTM too, so merged.
